src/components/button-toggle-group/button-toggle-group.stories.mdx

```diff
@@ -21,16 +21,14 @@ Press one of the buttons to make selection. ButtonToggleGroup component should b
 
 ## Quick Start
 
-To use the `ButtonToggleGroup` component you have to import two components: `<ButtonToggleGroup />` and `<ButtonToggle />`:
-
 ```javascript
 import ButtonToggle from "carbon-react/lib/components/button-toggle";
 import ButtonToggleGroup from "carbon-react/lib/components/button-toggle-group";
 ```
 
 ## Designer Notes
 
-- Always insert `ButtonToggle` Components inside the `ButtonToggleGroup`.
+- Only pass `ButtonToggle` components as children of `ButtonToggleGroup`, passing other anything else will result in an error.
```

Comment:
`passing other anything else` might need to be changed as it's not the best English 😆

Reply:
Indeed, I requested a change here (see below)